<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nct_Dialog</class>
 <widget class="QDialog" name="nct_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NCT Velocity</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Well</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="well_comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="log_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Log</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="sm_log_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Smoothed Log</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="sh_log_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="shale_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Shale Volume</string>
      </property>
     </widget>
     <widget class="QListWidget" name="horizon_listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>121</width>
        <height>171</height>
       </rect>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Horizons:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="plot_Button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>65</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plot_horizon_Button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>520</y>
        <width>65</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>NCT</string>
     </property>
     <widget class="QPushButton" name="add_Button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>570</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Save current result</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Saved NCT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="copy_Button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>520</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_name_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>new parameter name</string>
      </property>
      <property name="text">
       <string>nct_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_Button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw_Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>65</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fit_Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>65</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Fit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extrapolate_Button">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>65</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Extrapolate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_Button">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>65</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="a_lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>a:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>b:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="b_lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>start:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="start_lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>end:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="end_lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="two_points_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Use 2 points methods</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="two_points_groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="point1_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Point 1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="point2_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Point 2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="x_point1_lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="y_point1_lineEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="y_point2_lineEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="x_point2_lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="pick_checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Pick Points</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="nct_tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>121</width>
        <height>171</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>key</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>a</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>b</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
